File: Java/RAM.java
public class RAM
{
    private int capacity;
    private int price;

    //constructor
    public RAM(){}
    public RAM(int x, int y)
    {
        setCapacity(x);
        setPrice(y);
    }

    //capacity's methods
    public void setCapacity(int x)
    {
        this.capacity = x;
    }
    public int getCapacity()
    {
        return this.capacity;
    }

    //price's methods
    public void setPrice(int y)
    {
        this.price = y;
    }
    public int getPrice()
    {
        return this.price;
    }

}

File: CPP/RAM.cpp
#include <iostream>
#include <string>

using namespace std;

class RAM
{
    private:
        int capacity;
        int price;

    public:
        
        //constructors
        RAM(){}
        RAM(int x, int y)
        {
            setCapacity(x);
            setPrice(y);
        }

        //Capacity's methods
        void setCapacity(int x)
        {
            this->capacity = x;
        }
        int getCapacity()
        {
            return this->capacity;
        }

        //Price;s methods
        void setPrice(int y)
        {
            this->price = y;
        }
        int getPrice()
        {
            return this->price;
        }

        //destructor
        ~RAM(){}
};